code/modules/spells/aoe_turf/conjure/forcewall.dm

```diff
@@ -32,6 +32,41 @@
 	override_base = "grey"
 	hud_state = "mime_wall"
 
+/spell/targeted/mime_unwall
+	name = "Invisible un-wall"
+	desc = "Create an invisible un-wall on your location, an anomaly allowing the passage of all objects through anything on it"
+	school = "mime"
+	abbreviation = "FW"
+	user_type = USER_TYPE_OTHER
+	panel = "Mime"
+	specialization = SSOFFENSIVE
+
+	school = "mime"
+	duration = 300
+	charge_max = 300
+	cast_sound = null
+	cooldown_min = 2 SECONDS
+	spell_flags = WAIT_FOR_CLICK
+	range = 1
+	max_targets = 1
+	invocation_type = SpI_EMOTE
+	invocation = "mimes placing their hands on a flat surface, and pushing against it."
+
+	override_base = "grey"
+	hud_state = "mime_wall"
+
+/spell/targeted/mime_unwall/cast(var/list/targets, mob/user)
+	..()
+	for(var/atom/target in targets)
+		new /obj/effect/unwall_field(target.loc)
```

Comment:
I don't know much about spellcode. Are turfs valid targets? If so, I don't know what this would do.

Reply:
I've tested it, seems to not put one on them at all.

Reply:
That would be because it's placing it in the turf's area, I believe

Reply:
Fixed